docs/4.2/trustedclusters.md

```diff
@@ -311,6 +311,36 @@ and set `enabled` to "false", then update it:
 ```bsh
 $ tctl create --force cluster.yaml
 ```
+## Sharing Kubernetes groups between Trusted Clusters
+
+Below is an example of how to share a kubernetes group between trusted clusters. 
+
+In this example, we have a root trusted cluster with a role `root` and kubernetes groups:
+
+```yaml
+kubernetes_groups: ["system:masters"] 
+```
+SSH logins:
+
+```yaml
+logins: ["root"]
+```
+The leaf cluster can choose to map this `root` cluster to its own cluster. The `admin` cluster in the trusted cluster config:
+
+```yaml
+role_map:
+  - remote: "root"
+    local: [admin] 
+```
+The role `admin` of the leaf cluster can now be set up to use the root cluster role logins and `kubernetes_groups` using the following variables:
+
+```yaml
+logins: ['{{internal.logins}}']
+kubernetes_groups: ['{{internal.kubernetes_groups}}']
```

Comment:
One other thing, I have a feeling that these need to be in double quotes (`"{{internal.logins}}"`) rather than single quotes (`'`) for Teleport to interpolate them - I think at the very least, parsing via the web UI will fail with single quotes.

Reply:
Makes sense, thanks for clarifying that - addressed this now. I had seen some inconsistency in the rest of the docs, with this - I'll keep it in mind for future doc updates.